tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -405,7 +405,7 @@ public void AddPlaybackTransform(string recordingId, ResponseTransform transform
         #region common functions
         public string GetRecordingPath(string file)
         {
-            return Path.Join(RepoPath, "recordings", file);
+            return Path.Join(RepoPath, file);
```

Comment:
Does this mean we need to release again?

Reply:
Yeah. I'll be cutting another one for this.